test/clj/puppetlabs/trapperkeeper/testutils/webserver.clj

```diff
@@ -55,3 +58,25 @@
      ~port-var
      {}
      ~@body))
+
+(defn get-jetty-mbean-object-names
+  "TODO SPLAIN"
+  []
+  (let [mbeanserver (ManagementFactory/getPlatformMBeanServer)]
+    (.queryNames mbeanserver
+                 nil
+                 (ObjectName. "org.eclipse.jetty.jmx:type=mbeancontainer,*"))))
+
+(defn assert-clean-shutdown
+  "TODO SPLAIN"
+  [f]
+  (f)
+  ;; This sucks, because if this assertion fails, it will not give any clue as to
+  ;; which test caused it to fail beyond just the test namespace.  However, I tried
+  ;; several things (such as throwing an exception here rather than having an assertion),
+  ;; and nothing gave any better debugging info because the metadata about which
+  ;; test we're running and the call stack for the test function itself are simply
+  ;; not available from inside a fixture.  So I decided that even those these aren't
+  ;; super fun to debug, it was better than any other option in terms of enforcing
+  ;; that the tests clean up after themselves properly.
```

Comment:
I noticed that everywhere this is used, it is used as with `use-fixtures :each`. Since this won't provide any information beyond what test namespace the issue is in, it seems like it would provide the same amount of information if it was used with `use-fixtures :once` and perhaps not have as much overhead?